src/main/java/org/reactome/release/goupdate/GoUpdateStep.java

```diff
@@ -79,28 +93,32 @@ public void executeStep(Properties props) throws SQLException
 			{
 				Files.createDirectory(Paths.get("reports"));
 			}
-			duplicatePrinter = new CSVPrinter(Files.newBufferedWriter(Paths.get("reports/duplicate_GO_terms_"+dateString+".csv")), CSVFormat.DEFAULT.withAutoFlush(true).withHeader("DB_ID", "Accession", "GO type", "Before or After GO Update process?", "Number of referrers"));
-			reportOnDuplicateAccessions(adaptor, "BEFORE GO Update");
-			// Start a transaction. If that fails, the program will exit.
-			try
-			{
-				adaptor.startTransaction();
-			}
-			catch (TransactionsNotSupportedException e1)
+			try(BufferedWriter writer = Files.newBufferedWriter(Paths.get("reports/duplicate_GO_terms_"+dateString+".csv")))
```

Comment:
Not a big deal, but you could get rid of the slash here in the Paths.get() call so it will choose it choose its own file separator.  Saves you from having to hard code the slash and avoids system dependencies if you ever run on Windows :).

`Paths.get("reports", "duplicate_GO_terms" + dateString + ".csv")`